adlfs/spec.py

```diff
@@ -26,7 +26,7 @@
 from azure.storage.blob._shared.base_client import create_configuration
 from azure.storage.blob.aio import BlobServiceClient as AIOBlobServiceClient
 from azure.storage.blob.aio._list_blobs_helper import BlobPrefix
-from fsspec.asyn import AsyncFileSystem, get_loop, sync, sync_wrapper
+from fsspec.asyn import AsyncFileSystem, _get_batch_size, get_loop, sync, sync_wrapper
```

Comment:
Can we work in fsspec to expose a public version of this? Or at least get a promise that this isn’t going to change and break stuff here?

Reply:
https://github.com/fsspec/filesystem_spec/issues/1327